### PR TITLE
added alert

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

File: css/documentation/styles.css [hunk 1]
```diff
@@ -81,3 +82,9 @@ ul>li{
   justify-content: flex-start;
   padding-left: 20rem;
 }
+
+hr{
+  border: 1px solid;
+  width: 100%;
+  color: grey;
```
Comment (abhisheksahani-nova) [inline css/documentation/styles.css]: Use color variables for all color

**Review by abhisheksahani-nova — CHANGES_REQUESTED** [review]

**Review by anshulraheja — APPROVED** [review]
Suggestion: 
1. Add some space between the alert icon (span) and content (p)
2.  Use little dark shades for content (p) inside the colored alerts.

**Comment by dhruvin25799** [comment]
Looks good, just a minor suggestion to change the text color for colored alerts, having the same shade of color for text reduces visibility.

**Comment by Sahil25061999** [comment]
Overall good code, just a suggestion variable names can be more explicit and spacing between and the text and icons can be increased.

**Comment by ritikaawasthy** [comment]
> Looks good, just a minor suggestion to change the text color for colored alerts, having the same shade of color for text reduces visibility.

I've made the text color slightly darker for colored alerts to improve the visibility, let me know if this work.

**Comment by sumitk-singh** [comment]
overall it look good to me

**Pull request merged**